src/routes/api/index.js

```diff
@@ -2,13 +2,18 @@ import swaggerJsdoc from 'swagger-jsdoc';
 import swaggerUi from 'swagger-ui-express';
 import { Router } from 'express';
 import swaggerOptions from 'utils/api-specifications';
-import welcomeRoute from './welcome';
+import isLoggedIn from '../../utils/is-logged-in';
```

Comment:
```suggestion
import isLoggedIn from '../../utils/isLoggedIn';
```
Would you please consider to use camelCase naming convention

Reply:
thanks